src/tests/unit/tests/injected/visualization/drawer-provider.test.ts

```diff
@@ -1,5 +1,6 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { NavigatorUtils } from 'common/navigator-utils';
 import { IMock, Mock } from 'typemoq';
 import { BrowserAdapter } from '../../../../../common/browser-adapters/browser-adapter';
```

Comment:
probably a good time to update this import paths to non-relative ones

Reply:
I'd rather keep those to separate PRs